src/webgpu/api/operation/compute/basic.spec.ts

```diff
@@ -160,3 +160,76 @@ g.test('large_dispatch')
 
     dst.destroy();
   });
+
+g.test('buffer_binding_resource')
+  .desc(
+    `Validate the correctness of the buffer binding resource by filling the buffer with
+    testable data, clearing buffer in shader, and verifying the content of the whole buffer:
+  - covers the whole buffer
+  - covers the beginning of the buffer
+  - covers the end of the buffer
+  - covers neither the beginning nor the end of the buffer`
+  )
+  .paramsSubcasesOnly(u =>
+    u //
+      .combine('bindBufferResource', [false, true] as const)
```

Comment:
We could avoid generating the subcases with offset / size when bindBufferResource is true. Also bufferSize could be `extraBufferSize: [0, 8]` and the computation done in the test.